pkgs/applications/editors/emacs/elisp-packages/manual-packages.nix

```diff
@@ -17,6 +17,10 @@ lib.packagesFromDirectoryRecursive {
 
   elpaca = callPackage ./manual-packages/elpaca { inherit (pkgs) git; };
 
+  holo-layer = callPackage ./manual-packages/holo-layer {
+    inherit (pkgs) python3;
+  };
+
```

Comment:
```suggestion
```

Since there is no elisp package `python3`, there is no need for this.  Deleting this and renaming `holo-layer/default.nix` to `holo-layer/package.nix` should be enough.